{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "@pytest.mark.usefixtures(\"func_demo\")\n    @pytest.mark.usefixtures(\"class_demo\")\n    @pytest.mark.demo\n    def test_demo():\n        print(\"test\")\n>       assert False\nE       assert False\n\nTestCases\\LoginModuletest\\test_login_new.py:19: AssertionError"}, "start": 1591622883888, "stop": 1591622883888, "uuid": "6e3155b2-59f2-4410-9865-13b9f1983231", "historyId": "c602c8758b1d282fbf41304e01b6e025", "testCaseId": "395c50d9e7b550443350bc169249eaf8", "fullName": "TestCases.LoginModuletest.test_login_new#test_demo", "labels": [{"name": "tag", "value": "demo"}, {"name": "tag", "value": "@pytest.mark.usefixtures('class_demo')"}, {"name": "parentSuite", "value": "TestCases.LoginModuletest"}, {"name": "suite", "value": "test_login_new"}, {"name": "host", "value": "GC-HaiyuMa"}, {"name": "thread", "value": "11816-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.LoginModuletest.test_login_new"}]}